### PR TITLE
Change GetBootTime to work specificaly with FreeBSD

### PR DESCRIPTION
**Pull request by rikatz**
Signed-off-by: Ricardo Pchevuzinske Katz <ricardo.katz@gmail.com>

/kind cleanup

**What this PR does / why we need it**: Kubernetes have some specific files to work in FreeBSD, but it doesn't actually compile in FreeBSD because the mode to get the uptime used in GetBootTime differs from Linux (and FreeBSD does not have a sysinfo syscall).

This PR uses the following syscall: https://www.freebsd.org/cgi/man.cgi?query=clock_gettime&apropos=0&sektion=0&manpath=FreeBSD+12.2-RELEASE&arch=default&format=html

**Special notes for your reviewer**: 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

/sig node
/priority backlog

### CODE REVIEW

**Comment by ehashman** [comment]
@rikatz Can you please update the Linux test to remove the freebsd build, and add a similar unit test for freebsd? https://github.com/kubernetes/kubernetes/blob/12b4f7100df83169bfea4def594f1e8e6cf3a4a7/pkg/kubelet/util/bootime_util_linux_test.go#L1

**Comment by rikatz** [comment]
@ehashman right now :) 

I've also renamed bootime_util_linux_test.go to boottime_util_linux_test.go to reflect the tested package/file (boottime_util_linux.go

**Review by ehashman — APPROVED** [review]
/lgtm

seems fine to me, needs an approver to TAL

**Comment by rikatz** [comment]
Thanks! Do you want me to squash this into one commit only?

**Comment by ehashman** [comment]
These would be fine to squash since they're not logically separate, but generally k8s doesn't require squashing commits: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md

**Comment by rikatz** [comment]
Yup, I thought so (small commits right).

Squashed here, will seek for an approver in slack! Thank you for the review!

**Comment by rikatz** [comment]
/test pull-kubernetes-node-e2e-podutil

Triggering this to validate a non related prow job being worked with test-infra

**Comment by rikatz** [comment]
/pull-kubernetes-e2e-ubuntu-gce-network-policies
non related job just to test the network policies using podutils

**Comment by rikatz** [comment]
/test pull-kubernetes-e2e-ubuntu-gce-network-policies
non related job just to test the network policies using podutils

**Comment by rikatz** [comment]
/test pull-kubernetes-e2e-ubuntu-gce-network-policies

This is the lucky attempt

**Comment by ehashman** [comment]
that's an experimental optional slow test, why is it running? hrm. would need a repo admin to override.

**Comment by rikatz** [comment]
No no! I’ve run that test here because me and @aojea where looking into
migrating the job! No worries with this one :) my fault here

Em sex., 22 de jan. de 2021 às 19:49, Elana Hashman <
notifications@github.com> escreveu:

**Comment by rikatz** [comment]
/test pull-kubernetes-e2e-ubuntu-gce-network-policies

Testing again

@ehashman please disregard this test :)

**Comment by ehashman** [comment]
/triage accepted

@rikatz the test failure will block this PR from merging as an FYI

**Comment by rikatz** [comment]
> /triage accepted
> 
> @rikatz the test failure will block this PR from merging as an FYI

Thanks :)

I guessed, because it's optional (not Required) Prow would allow it to pass.

Anyway I still have to wait for some approval and I'm following sig-net folks re-writing of this test :) Let me check the job history to see if it does pass now.

Thank you very much! :)

**Pull request merged**
